lib/Net/Amazon/S3/Request/ListAllMyBuckets.pm

```diff
@@ -14,7 +14,7 @@ sub http_request {
     my $self    = shift;
     return $self->_build_http_request(
         use_virtual_host => 0,
-        authorization_method => 'Net::Amazon::S3::Signature::V2',
+        region => 'us-east-1',
```

Comment:
Is a comment about why region is needed useful for future?

Reply:
comment added.

Maybe there will be another issue related to region hostnames (eg: `s3.us-east-2.amazonaws.com`), see #59